eclipse-solargraph-plugin/src/main/java/io/github/pyvesb/eclipse_solargraph/utils/GemHelper.java

```diff
@@ -26,11 +26,17 @@
 
 public class GemHelper {
 
+	private static String buildPlatformCmd(String cmd, String gem) {
+		return String.format(
+			"gem %s -V -n \"%s\" \"%s\"",
```

Comment:
Is there any benefit to quoting the gem name?

Reply:
I just wanted to make sure, it was easy to add and it shouldn't harm. Don't know if GEM names with spaces are even possible, though. Remove or leave it?

Reply:
I don't _think_ it's possible even though the [naming conventions](https://guides.rubygems.org/name-your-gem/) don't seem to explicitly mention any constraints against spaces. Given the only two options at the moment are `solargraph` and `readapt`, I suggest removing them. 😄